### ISSUE TITLE
Add "Check Config" link in Configurator settings

### ISSUE THREAD

**Issue by GaryOkie** [danielperna84/hass-configurator#183]
"Check Config" would be a very useful shortcut link in the menu, just prior to "Restart HASS".

**Comment by danielperna84** [comment]
We once had that, but then the API at Home Assistant changed and it broke.

As there is a usable workaround by just having a Home Assistant tab open in a second tab, I don't see a big benefit. Especially considering that Home Assistants native capabilities to work with automations (and the configuration in general) gets better and better.

I'll tag this as _help wanted_ in case anyone want's to contribute.

**Comment by GaryOkie** [comment]
ok, thanks.   I thought it seemed like an obviously useful feature to have not already been included with all the other very useful shortcuts.

**Comment by danielperna84** [comment]
The difference compared to the other shortcuts is, that this actually produces output that has to be evaluated and displayed. So this takes more effort to implement.

**Comment by GaryOkie** [comment]
The "execute shell command" (href=#modal_exec_command) already provides a nice way to display responses from the executed command.   I tried to run _hassio homeassistant check_ as a shell command but it isn't found in the configurator docker.

If there was a way to run hassio via this tool, then perhaps this would be a relatively easy way to integrate the config check into the configurator menu as a 1-step click (eg: #modal_exec_hassiocheck)

**Comment by danielperna84** [comment]
Yes, it just requires an additional modal and some logic that fetches the response and displays it nicely. Back then the `homeassistant.check_config` service could be used for this. It won't be possible to use the command since in case of Docker or the hasssio-addon Home Assistant would live outside of the container. So the only way to do this is by using the API calls like I'm doing it for reloading configs currently. In fact, the API is the __only__ way the configurator should talk to Home Assistant (apart from reading the files from the filesystem).

**Comment by GaryOkie** [comment]
completely understood.  Thanks for the explanation!
